mlir/lib/Dialect/Linalg/Transforms/Transforms.cpp

```diff
@@ -381,11 +381,6 @@ FailureOr<LowerUnPackOpResult> linalg::lowerUnPack(RewriterBase &rewriter,
     return rewriter.notifyMatchFailure(unPackOp, "outer dims perm NYI");
 
   RankedTensorType packedTensorType = unPackOp.getSourceType();
```

Comment:
nit: I would move this line after `rewriter.setInsertionPoint(unPackOp);` as it is used further down now.